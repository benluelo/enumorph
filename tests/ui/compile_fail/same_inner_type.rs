#[derive(enumorph::Enumorph)]
enum Enum {
    A(u8),
    B { c: u8 },
}

fn main() {}
