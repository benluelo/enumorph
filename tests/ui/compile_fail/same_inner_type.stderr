error[E0119]: conflicting implementations of trait `TryFrom<Enum>` for type `u8`
 --> tests/ui/compile_fail/same_inner_type.rs:4:12
  |
3 |     A(u8),
  |       -- first implementation here
4 |     B { c: u8 },
  |            ^^ conflicting implementation for `u8`

error[E0119]: conflicting implementations of trait `From<u8>` for type `Enum`
 --> tests/ui/compile_fail/same_inner_type.rs:4:12
  |
3 |     A(u8),
  |       -- first implementation here
4 |     B { c: u8 },
  |            ^^ conflicting implementation for `Enum`
